assessments/2022-12-05-strings/code.js

```diff
@@ -0,0 +1,31 @@
+const upToX = (s) => {
+  return s.substring(0, s.indexOf('x'))
+}
+
+const charactersAround = (s, i) => {
+  return s[i - 1] + s[i + 1]
+}
+
+const middle = (s) => {
+  return s.substring(s.length / 4, s.length * 3 / 4)
+}
+
+const pair = (s1, s2) => {
+  return s1 + ' ' + 'and' + ' ' + s2
```

Comment:
The only nit I can pick with any of your code in this assessment is you could write this as:

```suggestion
  return s1 + ' and ' + s2
```